packages/eslint-config-airbnb/package.json

```diff
@@ -79,7 +79,7 @@
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-react": "^7.21.5",
-    "eslint-plugin-react-hooks": "^4 || ^3 || ^2.3.0 || ^1.7.0"
+    "eslint-plugin-react-hooks": "^4.2.0 || ^3 || ^2.3.0 || ^1.7.0"
```

Comment:
let's keep this at v4.0.1, so it's less likely to be a breaking change.

we'll also need to update the matching devDependencies line.